pkgs/development/python-modules/hydrus-api/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "hydrus-api";
+  version = "4.0.0";
```

Upstream says [`Python >= 3.9`](https://gitlab.com/cryzed/hydrus-api/-/blob/master/README.md#requirements).

```suggestion
  version = "4.0.0";
  format = "setuptools";

  disabled = pythonOlder "3.9";
```